### DISCUSSION TITLE
Germany: ZDF Full-HD Stream?

### DISCUSSION THREAD

**Discussion by Baumhausbanane123e** [iptv-org/community#1079] [Q&A]
Hello,
On iptv-org there is this streaming link for ZDF from Germany: https://zdf-hls-15.akamaized.net/hls/live/2016498/de/high/master.m3u8.
This one leads to a standard HD (720p) stream. I read somewhere else that if you replace _/high/_ with _/veryhigh/_ you get the full-hd stream. Did someone got this to work? I tried and VLC still says the video stream is 1280x720 (720p).

If anyone has got something to work or has an idead I'd be very glad!

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
